week2/2-thursday/homework/exercise6/answer.js

```diff
@@ -0,0 +1,13 @@
+// Cities 1
+var cities = [
+    { name: 'Los Angeles', temperature: 60.0},
+    { name: 'Atlanta', temperature: 52.0 },
+    { name: 'Detroit', temperature: 48.0 },
+    { name: 'New York', temperature: 80.0 }
+  ];
+
+  var coolerCities = cities.filter(function(el) {
+      return el.temperature < 70.0;
+  });
+
+  console.log(coolerCities);
```

Comment:
This is the same as the previous file. Wondering if there was a mixup on exercises.